src/datatype.rs

```diff
@@ -203,10 +203,7 @@ pub struct Code {
 }
 
 /// `CodeArg` represents a single argument of a [`Code`].
-// Possibly the literal arguments can themselves contain [ ] code blocks.
-// I'll have to test that.
-// A literal argument can be a string that starts with a (datatype) in front
-// of it, such as '(int32)0'.
+// TODO: Verify whether the literal arguments can themselves contain [ ] code blocks.
```

Comment:
I think we can say with some degree of caution that it can't

Reply:
This change will be gone after rebase:)